### PR TITLE
Update develop-in-the-cloud.md

### PR DESCRIPTION
**Pull request by mirnawong1**
adding clarification around capabilities of limit override in the ide. see [slack thread](https://dbt-labs.slack.com/archives/C02SRNY2EQ4/p1678220592398489?thread_ts=1678213967.104659&cid=C02SRNY2EQ4) for more info

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/get-started/develop-in-the-cloud.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Read more about IDE updates in the [dbt Cloud release notes](/docs/dbt-versions/
...
 | **IDE git button** | The git button in the IDE allows you to apply the concept of [version control](/docs/collaborate/git/version-control-basics) to your project. You can create or change branches, [resolve merge conflicts](/docs/collaborate/git/merge-conflicts), and execute git commands directly in the IDE. |
 | **Documentation** | You can generate and view your [project documentation](/docs/collaborate/build-and-view-your-docs) for your dbt project in real-time. You can inspect and verify what your project's documentation will look like before you deploy your changes to production. |
+| **Preview and Compile button** | You can run your code against your data platform by clicking the **Preview** or **Compile** button in the IDE. By default, the IDE will display 500 rows, but you can change this by adding a `limit your_number` command. For instance, `SELECT * FROM table limit 100` will return up to 100 rows. |
```
Comment (mirnawong1) [inline website/docs/docs/get-started/develop-in-the-cloud.md]:
```suggestion
| **Preview and Compile button** | You can run your code against your data platform by clicking the **Preview** or **Compile** button in the IDE. By default, the **Preview** button will display 500 rows, but you can change this by adding a `limit your_number` command. For instance, `SELECT * FROM table limit 100` will return up to 100 rows. |
```

File: website/docs/docs/get-started/develop-in-the-cloud.md [hunk 2]
```diff
@@ -47,6 +47,7 @@ Read more about IDE updates in the [dbt Cloud release notes](/docs/dbt-versions/
...
 | **IDE git button** | The git button in the IDE allows you to apply the concept of [version control](/docs/collaborate/git/version-control-basics) to your project. You can create or change branches, [resolve merge conflicts](/docs/collaborate/git/merge-conflicts), and execute git commands directly in the IDE. |
 | **Documentation** | You can generate and view your [project documentation](/docs/collaborate/build-and-view-your-docs) for your dbt project in real-time. You can inspect and verify what your project's documentation will look like before you deploy your changes to production. |
+| **Preview and Compile button** | You can run your code against your data platform by clicking the **Preview** or **Compile** button in the IDE. By default, the **Preview** button will display 500 rows, but you can change this by adding a `limit your_number` command. For instance, `SELECT * FROM table limit 100` will return up to 100 rows. |
```
Comment (mirnawong1) [inline website/docs/docs/get-started/develop-in-the-cloud.md]:
```suggestion
| **Preview and Compile button** | You can run your code against your data platform by clicking the **Preview** or **Compile** button in the IDE. By default, the **Preview** button will display 500 rows, but you can change this by writing `limit your_number` at the end of your SQL statement. For instance, `SELECT * FROM table limit 100` will return up to 100 rows.  Remember that you must write the `limit your_number` explicitly and cannot derive it from a macro. |
```

File: website/docs/docs/get-started/develop-in-the-cloud.md [hunk 3]
```diff
@@ -47,6 +47,7 @@ Read more about IDE updates in the [dbt Cloud release notes](/docs/dbt-versions/
...
 | **IDE git button** | The git button in the IDE allows you to apply the concept of [version control](/docs/collaborate/git/version-control-basics) to your project. You can create or change branches, [resolve merge conflicts](/docs/collaborate/git/merge-conflicts), and execute git commands directly in the IDE. |
 | **Documentation** | You can generate and view your [project documentation](/docs/collaborate/build-and-view-your-docs) for your dbt project in real-time. You can inspect and verify what your project's documentation will look like before you deploy your changes to production. |
+| **Preview and Compile button** | You can run your code against your data platform by clicking the **Preview** or **Compile** button in the IDE. By default, the **Preview** button will display 500 rows, but you can change this by writing `limit your_number` at the end of your SQL statement. For instance, `SELECT * FROM table limit 100` will return up to 100 rows.  Remember that you must write the `limit your_number` explicitly and cannot derive it from a macro. |
```
Comment (mirnawong1) [inline website/docs/docs/get-started/develop-in-the-cloud.md]:
```suggestion
| **Preview and Compile button** | You can run your code against your data platform by clicking the **Preview** or **Compile** button in the IDE. <br /> <br /> The **Preview** button will display 500 rows by default, but you can change this by adding `limit your_number` at the end of your SQL statement. For example, `SELECT * FROM table limit 100` will return up to 100 rows.  Remember that you must write the `limit your_number` explicitly and cannot derive it from a macro. |
```

Comment (matthewshaver) [reply website/docs/docs/get-started/develop-in-the-cloud.md]: The only additional feedback I have would be to change `writing` to something like `adding` and `For instance` to `For example`. Looks great! Thank you

Comment (mirnawong1) [reply website/docs/docs/get-started/develop-in-the-cloud.md]: ack, i honestly was going to change the 'for instance' but then i was like nevermind! always go w your gut lol

**Review by matthewshaver — APPROVED** [review]

File: website/docs/docs/get-started/develop-in-the-cloud.md [hunk 4]
```diff
@@ -47,6 +47,7 @@ Read more about IDE updates in the [dbt Cloud release notes](/docs/dbt-versions/
...
 | **IDE git button** | The git button in the IDE allows you to apply the concept of [version control](/docs/collaborate/git/version-control-basics) to your project. You can create or change branches, [resolve merge conflicts](/docs/collaborate/git/merge-conflicts), and execute git commands directly in the IDE. |
 | **Documentation** | You can generate and view your [project documentation](/docs/collaborate/build-and-view-your-docs) for your dbt project in real-time. You can inspect and verify what your project's documentation will look like before you deploy your changes to production. |
+| **Preview and Compile button** | You can run your code against your data platform by clicking the **Preview** or **Compile** button in the IDE. <br /> <br /> The **Preview** button will display 500 rows by default, but you can change this by adding `limit your_number` at the end of your SQL statement. For example, `SELECT * FROM table limit 100` will return up to 100 rows.  Remember that you must write the `limit your_number` explicitly and cannot derive it from a macro. |
```
Comment (davidharting) [inline website/docs/docs/get-started/develop-in-the-cloud.md:50]: I know this is already out in the wild - sorry for just now looking at it! 

One note is that `Compile` does not actually hit the data platform. Compile happens totally local within dbt.

Comment (mirnawong1) [reply website/docs/docs/get-started/develop-in-the-cloud.md:50]: ah thanks so much for clarifying that @davidharting and I'll get a pr up to clarify this 🙏

Comment (mirnawong1) [reply website/docs/docs/get-started/develop-in-the-cloud.md:50]: hey @davidharting , i opened up [this pr](https://github.com/dbt-labs/docs.getdbt.com/pull/3021) to clarify the above. would you be able to take a look to see if i captured it correctly? thank you!

Comment (davidharting) [reply website/docs/docs/get-started/develop-in-the-cloud.md:50]: thank you! Just gave that PR a ✅

**Pull request merged**